internal/pkg/externalplugins/tars/tars.go

```diff
@@ -214,7 +214,7 @@ func HandleAll(log *logrus.Entry, ghc githubClient, config *plugins.Configuratio
 		return nil
 	}
 	var buf bytes.Buffer
-	fmt.Fprint(&buf, "archived:false is:pr is:open")
+	fmt.Fprint(&buf, "archived:false is:pr is:open sort:created-asc")
```

Comment:
If the first PR was created a long time ago, will it also affect efficiency?

Reply:
Yes.